### DISCUSSION TITLE
Show persona card on hover shows different card for other users

### DISCUSSION THREAD

**Discussion by meklulose** [microsoft-search/pnp-modern-search#1534] [Q&A]
I am using PnP Modern Search v 4.3. When I enable the "show persona card on hover", it is showing different result when I am viewing my own card vs when I am viewing another user's card.  The following is when I am logged in as user test
![image](https://user-images.githubusercontent.com/81180944/142185206-124915d5-1bdf-472d-912b-a1a26bd6b744.png)
when I look at user test88 I am seeing less information 
![image](https://user-images.githubusercontent.com/81180944/142185350-15a7b7ae-3b3d-4595-8a60-7e2a932dce2b.png)

Is this an expected behavior? Ideally I want to be able to see the same information I am seeing in picture 1.

**Comment by wobba** [comment]
We have no control over this component and could be depending on what access scopes have been granted. Issues with the persona card should be logged at https://github.com/microsoftgraph/microsoft-graph-toolkit
